website/docs/reference/global-configs/legacy-behaviors.md

```diff
@@ -21,16 +21,17 @@ These flags can _only_ be set in `dbt_project.yml`. They configure behaviors clo
 ## Behaviors
 
 ### `source_freshness_run_project_hooks`
-- Should the 'dbt source freshness' command include [on-run-start / on-run-end hooks](/reference/project-configs/on-run-start-on-run-end)?
-- Planned introduction in v1.8, with default 'True'
-- Planned removal in v1.9
 
-If you have specific project `on-run-start` / `on-run-end` hooks that should not run before/after `source freshness` command, you can add a conditional check to those hooks:
-```yaml
-# dbt_project.yml
+If you have specific project [`on-run-start` / `on-run-end`](/reference/project-configs/on-run-start-on-run-end) hooks that should not run before/after `source freshness` command, you can add a conditional check to those hooks:
+
+<File name='dbt_project.yml'>
 
+```yaml
 on-run-start:
   - '{{ ... if flags.WHICH != 'freshness' }}'
 ```
+</File>
+
+You can configure the `dbt source freshness` command to include on-run-start / on-run-end hooks in v1.8 with a planned removal in v1.9. The default value is `True`.
```

Comment:
i like adding 'dbt version' initially so it's super clear

```suggestion
You can configure the `dbt source freshness` command to include on-run-start / on-run-end hooks in dbt version 1.8 with a planned removal in dbt version 1.9. The default value is `True`.
```

Reply:
We use this language throughout our docs. I think it's understood through historical context that v1.# refers to our dbt versions.